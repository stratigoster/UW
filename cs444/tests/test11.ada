-- no errors here
PACKAGE main IS
  default : INTEGER := abs (-10/(-3)) + 5 mod 3; -- this truncates :)
  s : STRING := "Batman";
  t : STRING := "Superman";
  b : BOOLEAN := s < t AND NOT (s = t) OR ELSE (default = default);
  a : BOOLEAN := s < t OR t <= s AND default >= default
                 OR ELSE s = t AND THEN t /= s;
END foo;

PACKAGE BODY main IS
BEGIN
  null;
END main;
