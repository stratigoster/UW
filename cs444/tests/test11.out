***ERROR***:2:9:	Package name declaration and closure are not the same

------
Compilation error
------
